homeassistant/components/hassio/__init__.py

```diff
@@ -145,7 +145,7 @@ async def async_install_addon(hass: HomeAssistantType, slug: str) -> dict:
     """
     hassio = hass.data[DOMAIN]
     command = f"/addons/{slug}/install"
-    return await hassio.send_command(command)
+    return await hassio.send_command(command, timeout=300)
```

Comment:
We may need even longer timeout here, since the installation can take a long time.

Reply:
I don't think we should add a timeout at all.

Reply:
```suggestion
    return await hassio.send_command(command, timeout=None)
```

Reply:
Yes. The caller of this function should decide the timeout, as each addon will be different.